packages/permission_handler/tizen/src/permission_handler_tizen_plugin.cc

```diff
@@ -10,16 +10,13 @@
 #include "service_manager.h"
 
 namespace {
-constexpr char kPermissionHandlerChannelName[] =
```

Comment:
Do you have a specific reason to remove it?

Reply:
Most of the plugins in this repo use string literals as channel names without declaring them as constants.

Reply:
I like this way!